extensions/amp-story/1.0/amp-story-tooltip.css

```diff
@@ -81,7 +70,7 @@
 
 .i-amphtml-hidden .i-amphtml-story-focused-state-layer-nav-button.i-amphtml-story-tooltip-nav-button-left {
   transform: translate3d(8px, 0, 0);
-  transition: transform 0.15s cubic-bezier(0.4, 0.0, 1, 1) !important;
+  transition: transform 0s linear .1s !important;
```

Comment:
Plz keep the material curves :))

Reply:
This is just removing the exit animation. (Delaying it .1s so that we don't see the jump in the transform).

Reply:
no position change for exit animation, so no curves here at all!!!!